scene/3d/vehicle_body_3d.cpp

```diff
@@ -225,6 +225,10 @@ bool VehicleWheel3D::is_in_contact() const {
 	return m_raycastInfo.m_isInContact;
 }
 
+Node3D *VehicleWheel3D::get_contact_body() const {	
```

Comment:
There seems to be a trailing tab here, the style checks would fail. While fixing it, might be worth addressing https://github.com/godotengine/godot/pull/55723/files#r765866806 too.

Reply:
Fixed these as well.